lua/octo/signs.lua

```diff
@@ -86,6 +86,10 @@ function M.render_signcolumn(bufnr)
     end_line = c["end_line"]
     M.place_signs(bufnr, start_line, end_line, c.dirty)
 
+    if c.owned then
+      M.place("octo_owned", bufnr, c.header_line)
+    end
```

Comment:
I think we shouldnt do the sign linehl approach and just change something in the header itself (such as bubblefy the author name). In that case we dont need to keep track of header lines and this should not be necessary

Reply:
Closing in regards of [this comment](https://github.com/pwntester/octo.nvim/pull/116#issuecomment-789578727).